doc/users/install.rst

```diff
@@ -119,7 +125,7 @@ Must Have
 Nibabel_ 1.0 - 1.4
```

Comment:
pip install nipype is here: https://github.com/nipy/nipype/pull/1485/files#diff-444fe9dbdb99f1db654e1abeebe704ebR40

The note above clarifies that I think. Or would you like to remove the section entirely?